src/api/dependenciesConfig.test.js

```diff
@@ -46,15 +48,57 @@ describe("api/dependenciesConfig", () => {
       expect(account.identity.name).toEqual(config.name);
       expect(account.identity.subtype).toEqual("PROJECT");
       expect(account.active).toBe(false);
+      expect(periods).toHaveLength(0);
+    });
+    it("can inject default period with weight", () => {
+      const ledger = new Ledger();
+      const autoInjectStartingPeriodWeight = 0.05;
+      const fakeNow = Date.now();
+
+      jest
+        .spyOn(global.Date, "now")
+        .mockImplementationOnce(() => fakeNow)
+        .mockImplementationOnce(() => fakeNow);
+
+      const config = {
+        name: n("foo"),
+        periods: [],
+        autoInjectStartingPeriodWeight,
+      };
+      const {periods} = ensureIdentityExists(config, ledger);
+      expect(periods).toHaveLength(1);
+      expect(periods[0].weight).toBe(autoInjectStartingPeriodWeight);
+      expect(fromISO(periods[0].startTime)).toBe(fakeNow);
+    });
+    it("does not inject default period if it already exists", () => {
```

Comment:
nit: `s/if it already exists/if any period already exists/`. (this test doesn't actually check the case where "it" (the default period) already exists, which is good, it tests the more general case where some other period exists.)